universum_log_collapser/e2e/universum_live_log_to_html.py

```diff
@@ -0,0 +1,53 @@
+#!/usr/bin/python3
+
+import os
+import subprocess
+import re
+
+
+def run_universum():
+    os.environ["PYTHONUNBUFFERED"] = "1"
```

Comment:
Maybe it'd be better to set environment for a subprocess instead of setting it to the parent process. It doesn't harm anything so far, it'd just be cleaner.